datafusion/core/Cargo.toml

```diff
@@ -40,8 +40,10 @@ name = "datafusion"
 path = "src/lib.rs"
 
 [features]
-# Used to enable the avro format
 nested_expressions = ["datafusion-functions-nested"]
+# This feature is deprecated. Use the `nested_expressions` feature instead.
+array_expressions = ["nested_expressions"]
```

Comment:
I have no better way to annotate a feature that is deprecated. Just leave a comment for it.

Reply:
I think this is ok -- thank you